qtconsole/frontend_widget.py

```diff
@@ -681,7 +684,8 @@ def append_stream(self, text):
         # Most consoles treat tabs as being 8 space characters. Convert tabs
         # to spaces so that output looks as expected regardless of this
         # widget's tab width.
-        text = text.expandtabs(8)
+        if self.expand_output_tabs:
+            text = text.expandtabs(8)
```

Comment:
@dalthviz, I think we should simply remove this line and leave tabs as tabs in qtconsole. That would give the same results for tabs in both qtconsole and the notebook, which I think is what most users would expect.